frontend/src/components/MeetingRoom/ToolbarButton/ToolbarButton.tsx

```diff
@@ -38,8 +38,8 @@ const ToolbarButton = forwardRef(function ToolbarButton(
         marginLeft: '0px',
         marginTop: '4px',
         marginRight: '12px',
-        width: isSmallViewPort ? '42px' : '48px',
-        height: isSmallViewPort ? '42px' : '48px',
+        width: isOverflowButton ? '35px' : '48px',
```

Comment:
The original reason for this ticket, we need to free some space for the `ScreenSharingButton` in case a desktop user uses a narrow/small browser window.

Reply:
ah yes, the original reason -  one line change - that blew up into a ~1000 lines of changes 😅